lib/Client.js

```diff
@@ -1394,11 +1394,13 @@ class Client extends EventEmitter {
     /**
     * Get a list of integrations for a guild
     * @arg {String} guildID The ID of the guild
+    * @arg {Object} [options={}] Additional options when getting guild Integration
+    * @arg {Object} [options.includeApplications=false] Whether to include bot and OAuth2 webhook integrations
     * @returns {Promise<GuildIntegration[]>}
     */
-    getGuildIntegrations(guildID) {
+    getGuildIntegrations(guildID, options = {}) {
         const guild = this.guilds.get(guildID);
-        return this.requestHandler.request("GET", Endpoints.GUILD_INTEGRATIONS(guildID), true).then((integrations) => integrations.map((integration) => new GuildIntegration(integration, guild)));
+        return this.requestHandler.request("GET", Endpoints.GUILD_INTEGRATIONS(guildID) + (options.includeApplications ? "?include_applications=true" : ""), true).then((integrations) => integrations.map((integration) => new GuildIntegration(integration, guild)));
```

Comment:
This should be done with Eris's data parameter despite only having one option. [For reference](https://github.com/abalabahaha/eris/blob/master/lib/Client.js#L1501) since we have things like this.